packages/transaction_signing_gateway/pubspec.yaml

```
@@ -25,12 +25,15 @@ dependencies:
       path: packages/cosmos_utils
       ref: main
 
+dependency_overrides:
+  analyzer: 3.3.1
+
 dev_dependencies:
   flutter_test:
     sdk: flutter
   lint: 1.8.1
   very_good_analysis: 2.4.0
-  mockito: 5.0.14
+  mocktail: ^0.3.0
```

let's stick with exact version for mocktail
```suggestion
  mocktail: 0.3.0
```